indico/modules/events/registration/client/js/form_submission/index.jsx

```diff
@@ -16,15 +16,27 @@ import reducers from './reducers';
 import RegistrationFormSubmission from './RegistrationFormSubmission';
 
 export default function setupRegformSetup(root) {
-  const {eventId, regformId, currency, management, moderated, userInfo, formData} = root.dataset;
+  const {
+    eventId,
+    regformId,
+    csrfToken,
+    currency,
+    management,
+    moderated,
+    userInfo,
+    submitUrl,
+    formData,
+  } = root.dataset;
 
   const initialData = {
     staticData: {
       eventId: parseInt(eventId, 10),
       regformId: parseInt(regformId, 10),
+      csrfToken,
```

Comment:
i'd move this below userInfo so the "basic" ones are all together (just for aesthetics)